test/integration/sandbox-libraries/pm.test.js

```diff
@@ -578,4 +578,322 @@ describe('sandbox library - pm api', function () {
+                                            raw: '{{URL}}',
+                                            host: [
+                                                '{{URL}}'
+                                            ]
+                                        }
+                                    },
+                                    response: [
+                                        {
+                                            name: 'New Request',
+                                            originalRequest: {
+                                                method: 'GET',
+                                                header: [],
+                                                url: {
+                                                    raw: 'localhost:3000',
+                                                    host: [
+                                                        'localhost'
+                                                    ],
+                                                    port: '3000'
+                                                }
+                                            },
+                                            _postman_previewlanguage: null,
+                                            header: null,
+                                            cookie: [],
+                                            body: null
+                                        }
+                                    ]
+                                }
+                            ],
+                            event: [
+                                {
+                                    listen: 'prerequest',
+                                    script: {
+                                        type: 'text/javascript',
+                                        exec: [
+                                            'console.log(\'collection pre\', pm.info.requestPath);'
+                                        ]
+                                    }
+                                },
+                                {
+                                    listen: 'test',
+                                    script: {
+                                        type: 'text/javascript',
+                                        exec: [
+                                            'console.log(\'collection test\', pm.info.requestPath);'
+                                        ]
+                                    }
+                                }
+                            ]
+                        }
+                    }, function (_err, results) {
+                        testrun = results;
+                        expect(testrun).to.be.ok;
+                        done();
+                    });
+                });
+
+                it(`should have path as correct path for a request nested inside folder when printed from
+                pre-request script of collection`, function () {
```

Comment:
What is the convention in this repo? Should test names contain a newline?
```suggestion
                it(
                  'should have path as correct path for a request nested inside folder when printed from pre-request script of collection',
                  function () {
```

Reply:
@codenirvana can guide here, not able to find more such instances.